WebViewControl/WebViewLoader.cs

```diff
@@ -36,13 +36,18 @@ internal static class WebViewLoader {
                 WindowlessRenderingEnabled = settings.OsrEnabled,
                 RemoteDebuggingPort = settings.GetRemoteDebuggingPort(),
                 UserAgent = settings.UserAgent
+             
             };
 
             var customSchemes = CustomSchemes.Select(s => new CustomScheme() { SchemeName = s, SchemeHandlerFactory = new SchemeHandlerFactory() }).ToArray();
 
             var customFlags = new[] {
                 // enable experimental feature flags
-                new KeyValuePair<string, string>("enable-experimental-web-platform-features", null)
+                new KeyValuePair<string, string>("enable-experimental-web-platform-features", null),
+
+                // OutOfBlinkCors moves CORS restriction handling from Blink to the NetworkService and is enabled by default since CEF 76.
+                // If enabled, it breaks the application of CORS restrictions for requests that are handled by us (e.g. importing fonts as external resources)
+                new KeyValuePair<string, string>("disable-features", "OutOfBlinkCors")
```

Comment:
this this still have any influence:
https://github.com/OutSystems/WebView/blob/d531c5c159c11b8cd22cd983caf910d163d2b105/WebViewControl/WebView.cs#L259

because the current semantics is that we used that setting to disable security restrictions... and if does not work.. we will have to review that property too

Reply:
I believe that setting is still useful for other security restrictions... Do you think we need to check if it still makes sense to have this?

Reply:
```
The CORS checks for custom schemes are now implemented in CEF (due to OutOfBlinkCors) and we don't check for the disable-web-security flag in that code.
```

Reply:
That's what made me think that `disable-web-security` is used in other places

Reply:
yeah thats my problem. I would like to have only one setting, but the thing is can we have this setting per browser instance? because disabling security for all is not that convinient

Reply:
We can have a setting per browser instance but when we would have to convert the IsSecurityDisabled webview property to a web view setting... because it cannot be changed once we pass the flags to the CefBrowserProcess

Reply:
Is that acceptable? It's probably the only way since `OutOfBlinkCors`is not configurable via chromium setting

Reply:
can you confirm that... because we might be able to change cefglue to accomplish that... it would be desired to have some instances with security off and others on